cfgov/data_research/urls.py

```diff
@@ -0,0 +1,22 @@
+from django.conf.urls import url
+
+from data_research.views import TimeSeriesData, TimeSeriesNational, MapData
+
+urlpatterns = [
+
+    url(r'^time-series/(?P<fips>\d{2,5})/?$',
+        TimeSeriesData.as_view(),
+        name='data_research_api_mortgage_timeseries'),
+    url(r'^time-series/national/?$',
+        TimeSeriesNational.as_view(),
+        name='data_research_api_mortgage_timeseries_national'),
+    url(r'^map-data/(?P<geo>counties)/(?P<year_month>\d{4}-\d{2})/?$',
+        MapData.as_view(),
+        name='data_research_api_mortgage_county_mapdata'),
+    url(r'^map-data/(?P<geo>metros)/(?P<year_month>\d{4}-\d{2})/?$',
+        MapData.as_view(),
+        name='data_research_api_mortgage_metro_mapdata'),
+    url(r'^map-data/(?P<geo>states)/(?P<year_month>\d{4}-\d{2})/?$',
+        MapData.as_view(),
+        name='data_research_api_mortgage_state_mapdata'),
```

Comment:
I'm not sure what you're trying to do here by repeating the same pattern multiple times. I don't think you can define multiple URL patterns with the same `name`, when resolving I believe Django will use the last one defined. And all of these patterns are the same with the exception of the name of the capture - could you just use `(?P<geo>counties|metros|states)` as a single pattern?